README.md

```diff
@@ -1,2 +1,170 @@
+  <ul>
+    <li>HTML</li>
+    <li>Javascript</li>
+    <li>CSS</li>
+    <li>React</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Make simple calculations**
+- **Read a random math-related quote**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+-[Live Demo Link](https://boosktore-react.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A [browser](https://www.google.com/search?q=what+is+a+browser&oq=what+is+a+browser&aqs=chrome..69i57.2748j0j1&sourceid=chrome&ie=UTF-8) of your choice
+- That you have set up Git on you desired computer
+- [NODEJS 16.10](https://nodejs.org/en) or higher
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd your-desired-folder
+  git clone git@github.com:veronica365/bookstore-react.git
+  open the repo folder `bookstore-react`
+```
+
+### Install
+
+Install this project dependencies with:
+
+```sh
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+```
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
+
+### Run tests
+
+To run tests to check the lint errors, run the following command:
+
+```sh
+  npx eslint . --fix
+```
+
+To run unit tests, run the following command:
+
+```sh
+  npm test
+```
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+
+
+### Deployment
+
+You can deploy this project using:
+
+```sh
+  npm run build
+
+```
+
+Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
+
+The build is minified and the filenames include the hashes.\
+Your app is ready to be deployed!
+
+See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [William](https://github.com/verocnica365)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] Add more tests and improve the application CSS styles
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/veronica365/bookstore-react/issues).
```

Comment:
- [ ] Kindly make sure to fix the `issues` link, it is broken and redirects to a 404 page